pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -229,6 +229,17 @@ func (b *BridgePodInterface) preparePodNetworkInterfaces() error {
 		return err
 	}
 
+	// turn TX offload checksum because it causes dhcp failures
+	if err := EthtoolTXOff(b.podInterfaceName); err != nil {
+		log.Log.Reason(err).Errorf("Failed to set tx offload for interface %s off", b.podInterfaceName)
+		return err
+	}
+
+	if err := EthtoolTXOff("vnet0"); err != nil {
```

Comment:
We need to be more dynamic. If we have multiple interfaces using multus or genie then the pod will have vnet[0...n] interfaces.

Reply:
@SchSeba Actually I have a better idea. The only thing we really need this for is for dhcp errors. I'll just move the code into `SingleClientDHCPServer` where we have access to `serverIface` that way I'm doing it for every interface that uses DHCP.